LayoutTests/js/dom/webidl-type-mapping-expected.txt

```diff
@@ -1244,6 +1249,10 @@ PASS converter.testSequenceRecord()['key'] is ['value', 'other value']
 PASS converter.setTestSequenceRecord({ 'Ā': ['value'] }) threw exception TypeError: Type error.
 converter.setTestSequenceRecord({ 'ÿ': ['value'] })
 PASS converter.testSequenceRecord()['ÿ'] is ['value']
+converter.setTestIndexRecord({ '1': true })
+FAIL converter.testSequenceRecord().hasOwnProperty('1') should be true. Was false.
```

Comment:
I can't figure out why this is failing...

Reply:
CC @szewai

Reply:
I'm not quite sure what I'm doing wrong so I created a follow-up bug <https://bugs.webkit.org/show_bug.cgi?id=284201> and removed this test case.